wheat_filtration/munge.py

```diff
@@ -5,6 +5,25 @@
 import nltk.data
 
 
+# make punctuation dictionary
```

Comment:
This is, like, right on the edge of where I'd say "make a function"...usually I like globals to be defined in one line, so if there's more logic to construct it, I'd make that one line a function call.

Reply:
is it a problem if you have a couple functions defined before you declare global variables? (i.e. the globals aren't declared directly under the imports)